--!strict


--> Modules -------------------------------------------------------------------------------------------
local Rsml = require("../src")
-------------------------------------------------------------------------------------------------------


--> Types ---------------------------------------------------------------------------------------------
-------------------------------------------------------------------------------------------------------


--> Variables -----------------------------------------------------------------------------------------
-------------------------------------------------------------------------------------------------------


--> Functions -----------------------------------------------------------------------------------------
-------------------------------------------------------------------------------------------------------



local tokens = Rsml.Tokenize("../project/shared/styles/Button.rsml")
local parsed = Rsml.Parse(tokens, "../project/shared/styles")

Rsml.OutputToModules(parsed, "../project/shared/styles/Button.rsml.luau")