--!strict


--> Modules -------------------------------------------------------------------------------------------
local Rsml = require("../src")
-------------------------------------------------------------------------------------------------------


--> Types ---------------------------------------------------------------------------------------------
-------------------------------------------------------------------------------------------------------


--> Variables -----------------------------------------------------------------------------------------
-------------------------------------------------------------------------------------------------------


--> Functions -----------------------------------------------------------------------------------------
-------------------------------------------------------------------------------------------------------


local path = "../project/shared/styles/Button.rsml"

local tokens = Rsml.Tokenize(path)
local parsed = Rsml.Parse(tokens, path:gsub("/[^/]+$", "") or path)
Rsml.OutputToModules(parsed, `{path}.luau`)