--!strict


--> Services ------------------------------------------------------------------------------------------
-------------------------------------------------------------------------------------------------------


--> Modules -------------------------------------------------------------------------------------------
local Tokenize = require("./tokenize")
local Parse = require("./parse")

local OutputToModules = require("./output/toModules")
-------------------------------------------------------------------------------------------------------


--> Types ---------------------------------------------------------------------------------------------
-------------------------------------------------------------------------------------------------------


--> Variables -----------------------------------------------------------------------------------------
-------------------------------------------------------------------------------------------------------


--> Functions -----------------------------------------------------------------------------------------
-------------------------------------------------------------------------------------------------------


return {
	Tokenize = Tokenize,
	Parse = Parse,

	OutputToModules = OutputToModules
}